src/DocumentationViewer.blp

```diff
@@ -0,0 +1,69 @@
+using Gtk 4.0;
+using Adw 1;
+using GObject 2.0;
+
+Window documentation_viewer{
+  title: "Documentation";
+  default-width: 1200;
+  default-height: 800;
+  hide-on-close: true;
+  titlebar:
+  Adw.HeaderBar {
+      Button back{
+        icon-name: "arrow1-left-symbolic";
+        tooltip-text: "Back";
+        styles ["flat"]
+      }
+      Button forward{
+        icon-name: "arrow1-right-symbolic";
+        tooltip-text: "Forward";
+        styles ["flat"]
+      }
+      ToggleButton sidebar_button{
+        active: false;
+        icon-name: "dock-left-symbolic";
+        tooltip-text: "Toggle Sidepanel";
+      }
+      ToggleButton search_button {
+        icon-name: "loupe-large-symbolic";
+        tooltip-text: "Search";
+      }
```

Comment:
I don't think we need both and opening the sidebar should open the search. But let's iterate on the UI in a follow up.